code/_onclick/ai.dm

```diff
@@ -223,6 +223,13 @@
 /obj/machinery/ai_slipper/AIAltClick() //Dispenses liquid if on
 	Activate()
 
+/obj/machinery/firealarm/AICtrlClick()
+	if(enabled)
+		reset()
+	else
+		alarm()
+	return
```

Comment:
The others have `Topic` stuff, but this seems to work so... 🤷

Reply:
You can leave off the return here. The proc will already return regardless.

Reply:
Most of the other procs in that file have `return` at the end, so I just followed what they were doing.